<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>506</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Verdana</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>чатик без калькулятора</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3,1,3,1,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="nickLabel">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Ник:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nicknameEdit">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Johny</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="portLabel">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string> Порт:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="portNumEdit">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1248</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="enterChatButton">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>вход</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="changeButton">
        <property name="text">
         <string>Сменить ник</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <layout class="QGridLayout" name="gridLayout" columnstretch="7,3">
        <item row="4" column="0">
         <widget class="QLineEdit" name="messageEdit"/>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="sendButton">
          <property name="text">
           <string>отправить</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QTextBrowser" name="log">
          <property name="styleSheet">
           <string notr="true">background-color: rgba(0, 0, 0, 0)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>онлайн:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" rowspan="3">
         <widget class="QPlainTextEdit" name="plainTextEdit"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>чат</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QListWidget" name="onlineList"/>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>лог</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>refreshOnlineList()</slot>
 </slots>
</ui>
